### PR TITLE
Add radiation card

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes # 171305605

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

image

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by nerdlet** [comment]
@chegejames Ping

File: src/components/Landing/Hero.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function Hero({ classes }) {
       <Grid item xs={6}>
         <Typography variant="h4" className={classes.intro}>
           We are here to give you actionable information about the quality of
-          your city&apos;s air, water and sound.
+          your city&apos;s air, water, sound and radiation level.
```
Comment (chegejames) [inline src/components/Landing/Hero.js]: @nerdlet Posts deployment, let us run this past editorial for copy editing.

Comment (nerdlet) [reply src/components/Landing/Hero.js]: @chegejames  😞 We already have a feedback doc for this. You need to put this in there  and flag      it for editing

Comment (nerdlet) [reply src/components/Landing/Hero.js]: @chegejames Edited the text.

**Pull request merged**